scanner/updater/manual/data.go

```diff
@@ -0,0 +1,24 @@
+package manual
+
+import (
+	"github.com/quay/claircore"
+)
+
+var manuallyEnrichedVulns = []*claircore.Vulnerability{
```

Comment:
is this just an example? If so, let's maybe throw it into an example_test.go file: https://go.dev/blog/examples